app/controllers/coronavirus_form/address_lookup_controller.rb

```diff
@@ -18,6 +18,7 @@ def submit
 
     address = JSON.parse(params[:address]).to_h
     session[:support_address] = helpers.convert_address(address)
+    session.delete(:postcode)
```

Comment:
Why is this being deleted?

Reply:
This is added [here](https://github.com/alphagov/govuk-coronavirus-vulnerable-people-form/blob/290adc4f9e2def85ab7ca32c1c7ba3c105c829d1/app/controllers/coronavirus_form/postcode_lookup_controller.rb#L26) but only for the address lookup. This is a duplicate of the `support_address.postcode` field.